Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ubuntu:latest AS builder
+RUN apt-get update
+RUN apt-get install -y --no-install-recommends build-essential git cmake libboost-dev libpq-dev
+RUN apt-get clean
+
+WORKDIR /builder/alicia-server
+ADD . .
+RUN git submodule update --init --recursive
```

Comment:
If we're initializing the submodules as a part of the build step, shouldn't we also clone the repository? 
My train of though behind this is that if repository is already provided, it should be already a valid one.

Reply:
Disregard, I somehow failed to realize that this is within the repository itself.